Docs: Settings updates at runtime (#34169)
grafana/grafana: docs/sources/http_api/admin.md

  }
}
```

## Update settings

`PUT /api/admin/settings`

> **Note:** Available in Grafana Enterprise v8.0+.

Updates / removes and reloads database settings. You must provide either `updates`, `removals` or both.

> It currently only supports changes on `auth.saml` section.

Reviewer: ```suggestion
This endpoint only supports changes to `auth.saml` configuration.
```
